frontend/src/components/StockNews/StockNews.jsx

```diff
@@ -1,105 +1,97 @@
 import "./StockNews.css";
 import { useState, useEffect, useContext } from "react";
 import axios from "axios";
-import { formatDate, firstWord } from "../../utils.js";
+import { formatDate, getMarketNewsUrl, getStockNewsUrl } from "../../utils.js";
 import { UserContext } from "../App/App";
+import Card from "react-bootstrap/Card";
+import Dropdown from "react-bootstrap/Dropdown";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import Button from "react-bootstrap/Button";
 
 export default function StockNews({ stocks }) {
     const [stockNews, setStockNews] = useState([]);
     const [currentStock, setCurrentStock] = useState(null);
     const { setErrorMessage, setLoading } = useContext(UserContext);
+    const [articlesToShow, setArticlesToShow] = useState(5);
```

Comment:
this is what would be called a ["magic number"](https://codehs.com/glossary/term/14). It's fine to set this to 5, but make it a constant so it's clearer where it is defined